src/components/panes/ImportLogPane.jsx

```diff
@@ -0,0 +1,135 @@
+                    },
+                    {
+                        name: 'imported',
+                        msgId: 'panes.importLog.report.imported',
+                        msgNumbers: { imported: [log.report.imported] },
+                    },
+                    {
+                        name: 'created',
+                        msgId: 'panes.importLog.report.created',
+                        msgNumbers: { created: [log.report.created] },
+                    },
+                    {
+                        name: 'updated',
+                        msgId: 'panes.importLog.report.updated',
+                        msgNumbers: { updated: [log.report.updated] },
+                    },
+                    {
+                        name: 'tagged',
+                        msgId: 'panes.importLog.report.tagged',
+                        msgNumbers: { tagged: [log.report.tagged] },
+                    },
+                    {
+                        name: 'taggings',
+                        msgId: 'panes.importLog.report.taggings',
+                        msgNumbers: { taggings: [log.report.taggings] },
+                    },
+                ];
+                infoListData = infoListData.concat(completedData);
+            } else if(log.status == 'error') {
+                infoListData.push(
+                    {
+                        name: 'error',
+                        msgId: 'panes.importLog.error',
+                        value: log.error
+                    },
+                );
+            }
+
+            /*
+            const avatar = log.imported_by ? (
+                <div className="ImportLogPane-imported_by">
+                    <Avatar key="avatar" ref="avatar" person={ log.imported_by } />
+                    <div className="ImportLogPane-imported_by-info"
+                        { log.imported_by.first_name + ' ' + log.imported_by.last_name }
+                    </div>
+                </div>)
+                : <Msg tagName="p" id="panes.importLog.noPerson" />;
+                */
+            let importedby;
+            if(log.imported_by) {
+                const name = log.imported_by.first_name + ' ' + log.imported_by.last_name;
+                const email = log.imported_by.email;
+                importedby = <div key="importedby" className="ImportLogPane-importedby">
+                        <Msg tagName="h3" id="panes.importLog.imported_by" />
+                        <Avatar ref="avatar" person={ log.imported_by } />
+                        <div className="ImportLogPane-personInfo">
+                            <span className="ImportLogPane-name">
+                                { name }
+                            </span>
+                            <span className="ImportLogPane-email">
+                                { email }
+                            </span>
+                        </div>
+                    </div>;
```

Comment:
It would be nice if this would link to the person pane. Maybe use an existing component (e.g. the ones used in `ActionPane` or `CallAssignmentPane`) for the person which should already support clicking including hover effects et c.